components/bsx/Create/Create.vue

```diff
@@ -4,18 +4,14 @@
   </div>
 </template>
 
```

Comment:
Could pls check this GitHub actions error ?

Reply:
which error?

Reply:
<img width="1473" alt="image" src="https://user-images.githubusercontent.com/31397967/213629296-3306f3f8-f553-4a7b-b4e5-13e9ecff5f0c.png">

Reply:
oh, let me see

Reply:
renamed the component